.claude/hooks/session-start.sh

```
@@ -0,0 +1,34 @@
+# Non-fatal on network failure: a transient github.com outage shouldn't block
+# the whole session. We log and continue.
+
+set -uo pipefail
+
+# Only act in the remote (Claude Code on the web) environment. Local devs
+# manage their own submodules.
+if [ "${CLAUDE_CODE_REMOTE:-}" != "true" ]; then
+  exit 0
+fi
+
+cd "${CLAUDE_PROJECT_DIR:-$(pwd)}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)